systemd/coreos-installer-reboot.service

```diff
@@ -8,3 +8,5 @@ ConditionPathExists=/run/coreos-installer-reboot
 [Service]
 Type=simple
 ExecStart=/usr/bin/systemctl --no-block reboot
+StandardOutput=kmsg+console
+StandardError=kmsg+console
```

Comment:
Is this just to be consistent, or did you actually encounter an issue with the service?

Reply:
I think there was a case where something wasn't working and I had to add this to find out why

Reply:
ahh yes. I think it was fab2fcc